conftest.py

```diff
@@ -69,14 +73,37 @@ def filter_response(response):
     return response
 
 
+def remove_host_request(request):
+    request.uri = re.sub(r"(?<=://)[^/]+", "example.com", request.uri)
```

Comment:
Could you perhaps document what this regex does? It boggles the mind :)

Wouldn't something like `re.sub(r"https?://(.+)", "https://example.com", request.uri` make more sense and be easier to read?

Also since this RE will be used for every request, might as well compile it

Reply:
I'm replacing the current pattern with `(https?://)[^/]+` for better readability, the option of using `(?<=://)` (positive lookbehind) was to match and keep the same protocol used in the original request but I do't think its really needed so I've removed it... About the last part I think I should keep the `[^/]+` so it matches everything but stop matching after the 1st slash it find. Something like `(.+)` would also replace the context, e.g.:

```
>>> re.sub("(https?://)(.+)", "https://example.com", "https://github.com/RedHatProductSecurity/osidb/pull/415")
'https://example.com'

>>> re.sub("(https?://)[^/]+", "https://example.com", "https://github.com/RedHatProductSecurity/osidb/pull/415")
'https://example.com/RedHatProductSecurity/osidb/pull/415'
```

Please let me know if this is a valid solution or if there is still room for improvement.